lib/engine_cart/tasks/engine_cart.rake

```diff
@@ -32,9 +31,9 @@ namespace :engine_cart do
           "_#{Gem.loaded_specs["rails"].version}_"
         end
 
-        Bundler.with_clean_env do
-          `rails #{version} new internal --skip-spring #{EngineCart.rails_options} #{"-m #{EngineCart.template}" if EngineCart.template}`
-        end
+        # Using bundle exec rake engine_cart:generate
+        # Rails might not be installed at system level.  What would clean env afford us here?
+        `rails #{version} new internal --skip-spring #{EngineCart.rails_options} #{"-m #{EngineCart.template}" if EngineCart.template}`
```

Comment:
I'm pretty sure we need `with_clean_env` here to avoid funky gem dependency conflicts. In 204ca38402609d98f635390a1262349230db5d11, I've tried to update this so it will call the appropriate rails executable.